IN-global_conf.json

```diff
@@ -1,78 +1,79 @@
 {
 	"SX1301_conf": {
 		"lorawan_public": true,
-		"clksrc": 1,
+		"clksrc": 0,
 		"clksrc_desc": "radio_1 provides clock to concentrator for most devices except MultiTech. For MultiTech set to 0.",
-		"antenna_gain": 0,
+		"antenna_gain": 1.4,
```

Comment:
Default? I don't know. I have asked MultiTech about this. Haven't got a real answer; and unfortunately I'm not knowledgeable enough about the SEMTECH 13xx to be able to reverse engineer it. There are dB changes (RSSI, etc) all through the file, and I don't know how they interact. It looks consistently like 4dB of difference.